scripts/scil_convert_sh_basis.py

```diff
@@ -25,9 +26,15 @@ def _build_arg_parser():
 
     p.add_argument('in_sh',
                    help='Input SH filename. (nii or nii.gz)')
-
     p.add_argument('out_sh',
                    help='Output SH filename. (nii or nii.gz)')
+    
+    p.add_argument('--is_input_not_legacy', action='store_true',
```

Comment:
Definitively a detail but I would like better this way
```
is_input_not_legacy -> in_sh_is_legacy (since we know it is (or not) and I'm not found of double negative)
is_output_not_legacy -> out_sh_legacy
```